content/en/docs/reference/kubectl/_index.md

```diff
@@ -101,7 +101,7 @@ then kubectl assumes it is running in your cluster. The kubectl tool looks up th
 namespace of that ServiceAccount (this is the same as the namespace of the Pod)
 and acts against that namespace. This is different from what happens outside of a
 cluster; when kubectl runs outside a cluster and you don't specify a namespace,
-the kubectl command acts against the `default` namespace.
+the kubectl command acts against the current-context's namespace.
```

Comment:
or ` the kubectl command acts against the namespace set for the current context in your client configuration.`

let me know WDYT?

Reply:
I agree with your second suggestion @akankshakumari393 . I think it is best to leave out the word "default" because someone might think you mean the literal `default` namespace.

Reply:
> the kubectl command acts against the namespace set for the current context in your client configuration

To me, this wording is unclear on what happens when your current context does not specify a namespace. It might even imply that contexts must specify a namespace. If we need to clarify the situation, I'm in favor of extra sentences to make the situation clear.

### Alternative
If we have a suitable existing task page about writing a kubeconfig, we could link there rather than go into lots of detail.

Reply:
I have modified the line and added few detail. PTAL